starter/source/elements/initia/hm_read_inistate_d00.F

```diff
@@ -2250,7 +2250,7 @@ SUBROUTINE HM_READ_INISTATE_D00 (
                         SIZE = NPG
                         CALL HM_GET_FLOAT_ARRAY('sigma_X'  ,TMPVAL1,SIZE,J,IS_AVAILABLE,LSUBMODEL,UNITAB)
                         CALL HM_GET_FLOAT_ARRAY('sigma_Y'  ,TMPVAL2,SIZE,J,IS_AVAILABLE,LSUBMODEL,UNITAB)
-                        CALL HM_GET_FLOAT_ARRAY('sigma_Z'  ,TMPVAL3,SIZE,J,IS_AVAILABLE,LSUBMODEL,UNITAB)
+                       !! CALL HM_GET_FLOAT_ARRAY('sigma_Z'  ,TMPVAL3,SIZE,J,IS_AVAILABLE,LSUBMODEL,UNITAB)
```

Comment:
Hi  @SaidMamouri said you don't fill TMPVAL3 value anymore , but it is used after , line 2270

Sébastien

Reply:
Thx Seb.

Reply:
You're welcome , it is now ok , Thanks a lot @SaidMamouri